pkg/api/v1/conversion.go

```diff
@@ -125,7 +125,9 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Namespace",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "status.phase":
+			case "name":
```

Comment:
don't remove "name" from objects where it already existed, but don't add it to new objects